changelogs/8.7.asciidoc

```diff
@@ -1,8 +1,6 @@
 [[release-notes-8.7]]
 == APM version 8.7
 
-https://github.com/elastic/apm-server/compare/8.6\...8.7[View commits]
```

Comment:
I see that you are deleting branch diff from 8.7, but keeping it in 8.5, 8.6 etc. Is this expected?